dashboard/test/models/pd/application/teacher_application_test.rb

```diff
@@ -647,6 +650,7 @@ class TeacherApplicationTest < ActiveSupport::TestCase
 
       application_hash = build :pd_teacher_application_hash,
         program: Pd::Application::TeacherApplication::PROGRAMS[:csp],
+        will_teach: 'Yes',
```

Comment:
I did it because I don't like using magic variables like this but I clearly missed this one 🤦‍♀️ This PR was a bit of test failure whack-a-mole so I think I got frustrated at some point haha

Reply:
Oh gotcha gotcha, yes that totally makes sense :)